bin/compile

```diff
@@ -145,7 +150,6 @@ if [ ! -z "$env_dir" ]
 then
     mkdir -p "$env_dir"
     env_dir=$(cd "$env_dir/" && pwd)
-    ln -sfn $build /app/code
```

Comment:
I think this dates back to f9f79478fa74d74fb92bfdceea11080bccbea872 and 43f7b5f285ffff6ebc71f435051ea49420c1d855.